subscribe.go

```diff
@@ -305,3 +305,11 @@ func (ls *legSubscriber) onSyncEvent(c cid.Cid, out chan cid.Cid, ulOnce *sync.O
 		}
 	}
 }
+
+// getLatestSync gets the latest synced link.
+// This function is safe to call from multiple goruties and exposed for testing purposes only.
```

Comment:
```suggestion
// This function is safe to call from multiple goroutines and exposed for testing purposes only.
```

Unless you're trying to coin a new term :P